canonical-web-and-design/maas-ui
# Display the user data in the edit form.

Done:
- Load and display user data in the edit form. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1356.
- See a password length. Fixes: #47.
- Update the Formik fields to reduce duplication and be testable.
- Set unique ids on form fields. Fixes: #72

QA:
- Navigate to the user list.
- Click on the edit icon for a user.
- You should get the edit form with the user's data.
